openstf/model_selection/model_selection.py

```diff
@@ -1,87 +1,254 @@
 # SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
 #
 # SPDX-License-Identifier: MPL-2.0
-from typing import List, Tuple
-from datetime import timedelta
-
 import numpy as np
 import pandas as pd
-import structlog
+import secrets
+
+from datetime import datetime, timedelta
+from dateutil.parser import parse
+
+from typing import List
+
+AMOUNT_DAY = 96  # Duration of the periods (in T-15) that are in a day (default = 96)
+PERIOD_TIMEDELTA = 1  # Duration of the periods (in days) that will be sampled as validation data for each split.
+MAX_QUANTILE = 0.95
+MIN_QUANTILE = 0.05
+
+
+def find_min_max_peaks(data: pd.DataFrame) -> (List[float], List[float]):
+    """
+    Retrieve minimum and maximum quantiles for a day (start till end)
+
+    Args:
+        data (pandas.DataFrame): Clean data with features
+
+    Returns:
+        min_peaks (list:float): Minimum peaks of a day
+        max_peaks (list:float): Maximum peaks of a day
+
+    """
+
+    try:
+        min_peaks = data["days"][:][data["load"] < data["load"].quantile(MIN_QUANTILE)]
```

Comment:
I have few points here:
 - It is best to avoid these hard coded column names and when you do use them it is better to make a check.
 - I cannot find where the "days" column is added to data.
 - It is nice to add a short comment about what this line does